package.json

```diff
@@ -37,7 +38,8 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-hook-form": "^7.45.1",
-    "yup": "^1.2.0"
+    "yup": "^1.2.0",
+    "zod": "^3.22.2"
```

Comment:
Treng vi to bibliotek for validering av data? Kan vi ikkje berre bruke yup?

Reply:
Glemte at vi hadde yup. Fikset i siste commit 👍